install/export-bystro-libs.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [[ -n "$1" ]]
+then
+  PROFILE=$1;
+else
+  PROFILE=~/.bash_profile;
+fi
+
+DIR=$(pwd)
+
+echo -e "\n\nExporting $DIR/lib and $DIR/bin to $PROFILE\n";
+
+if ! cat $PROFILE | grep "$DIR/bin"; then
+  (echo "" ; echo 'export PERL5LIB=$PERL5LIB:'$DIR'/lib') | sudo tee -a $PROFILE
+  # Not 100% sure why this is necessary; something still wonky during perlbrew install
+  (echo "" ; echo 'export PATH=$PATH:'$DIR'/bin') | sudo tee -a $PROFILE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/akotlar/bystro/pullRequest?prid=4888220)